stage0/library

```diff
@@ -0,0 +1 @@
+../src/stage0
```

Comment:
@leodemoura `stage0/library` and `stage0/src` are just symlinks here to avoid merge conflicts and huge diff lists; you should call `update-stage0` before trying a parallel change, and after this commit is merged into `master`. Then we can remove `src/stage0`.